src/Serilog.Sinks.MSSqlServer/Serilog.Sinks.MSSqlServer.csproj

```diff
@@ -47,11 +47,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Serilog, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Serilog.1.5.14\lib\net45\Serilog.dll</HintPath>
+      <HintPath>..\..\..\..\src\packages\Serilog.1.5.14\lib\net45\Serilog.dll</HintPath>
```

Comment:
Ln 50+54 changes is a result of us having included the sink as a submodule in an existing solution. I'll be happy to make a new PR without this change as it should not go into the final merge should you choose to accept the PR.